pkg/coordinator/coordinator.go

```diff
@@ -28,6 +29,12 @@ import (
 	"github.com/smartcontractkit/ocr2keepers/pkg/util"
 )
 
+type Coordinator interface {
+	Filter() func(types.UpkeepKey) bool
```

Comment:
We should resolve what we're doing with this function. I think we can probably merge all the PRs together to move forward a bit, but filtering keys should be the next focus.